cmd/main.go

```diff
@@ -49,6 +49,18 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+// nonLeaderCertWatcher is a wrapper around certwatcher.CertWatcher
+// that doesn't require leader election.
+type nonLeaderCertWatcher struct {
```

Comment:
```suggestion
type customCertWatcher struct {
```
explain the objective in comments